cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -1059,7 +1097,9 @@ def test_validate_from_json_wrong_inputs(self):
         self.assertNotEqual(test_model['days_early_for_beta']['value'], "supposed to be an integer",
                             'days_early_for beta should not be updated to a wrong value')
 
-    def test_correct_http_status(self):
+    @patch('cms.djangoapps.contentstore.views.course.IgnoreMobileAvailableFlagConfig.is_enabled')
```

Comment:
A small nit here in all other places where we just need to return `False`: `@patch('cms.djangoapps.contentstore.views.course.IgnoreMobileAvailableFlagConfig.is_enabled')
, return_value=False`